css/events.css

```diff
@@ -29,43 +29,42 @@
   margin: 3px auto 3px;
   text-align: center;
   /* color: rgb(228, 58, 99); */
-  color: #FF3366;
+  color: #ff3366;
 }
 
-@media  screen and (max-width: 640px){
-  .event-title{
+@media screen and (max-width: 640px) {
+  .event-title {
     font-size: 2.7em;
   }
 }
-@media  screen and (max-width: 440px){
-  .event-title{
+@media screen and (max-width: 440px) {
+  .event-title {
     font-size: 1.95em;
     font-weight: 700;
   }
 }
 
-@media screen and (min-width: 640px){
-  .event-title{
+@media screen and (min-width: 640px) {
+  .event-title {
     font-size: 3.3em;
   }
 }
 
 .event-date-time {
-  margin: 0 auto;
-  text-align: center;
-  font-weight: 400;
-  font-size: 1.07em;
+  font-weight: 600;
+  font-size: 1.1em;
 }
 
 .session-info {
   margin: 0 30px 50px;
   max-width: 700px;
-  border: 2px solid grey;
-  border-radius: 5px;
-  padding: 25px;
+  border-radius: 10px;
+  padding: 40px;
   display: block;
-  box-shadow: 0 20px 30px 0px rgba(228, 58, 99, 0.2);
   background: #fff;
+  -webkit-box-shadow: 2px 2px 18px 0px rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 2px 2px 18px 0px rgba(0, 0, 0, 0.75);
+  box-shadow: 2px 2px 18px 0px rgba(0, 0, 0, 0.75);
```

Comment:
I don't think we need so many vendor prefixes @tejasberi. Please confirm the ones we should fly in

Reply:
Since this is a plane HTML-CSS project, Please confirm if there is a auto-prefixer in the build/deploy cycle for this project. If not then its better to have those many vendor prefixes.

Reply:
@tejasberi We don't need prefixes for box-shadow for most browsers nowadays: https://caniuse.com/#search=box-shadow

Reply:
done